Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -56,6 +75,8 @@ class UserScriptManager {
             }
         }
 
+        tab.webView?.configuration.userContentController.addUserScript(FullscreenHelperScript!)
```

Comment:
@garvankeeley Think this is the right place to put it or shall we refactor the code? Any suggestion?